pyproject.toml

```diff
@@ -0,0 +1,549 @@
+[tool.pydocstyle]
```

Comment:
You can also move setup.py configuration inside. This is not involved. Here is the example https://github.com/pymc-devs/pytensor/blob/main/pyproject.toml

Reply:
Addressed in [b0f2e8b](https://github.com/geoopt/geoopt/pull/201/commits/b0f2e8b8c04fcc0505d629530da370a03c370591)